lib/babe/babe_integration_test.go

```diff
@@ -257,17 +266,15 @@ func TestService_GetAuthorityIndex(t *testing.T) {
 }
 
 func TestStartAndStop(t *testing.T) {
-	serviceConfig := ServiceConfig{}
-	bs := createTestService(t, serviceConfig)
+	bs := createTestService(t, ServiceConfig{})
```

Comment:
None of this comment makes sense to me.

> createTestService(t, ServiceConfig{}) is uglier.

Ugly and beautiful is very subjective. What you are finding ugly here, is prettier for me.

there is an empty `serviceConfig` created only to pass in the very next line, which does not make any sense to me. Here you are advocating creation of variable which is used only once. 

While on the other side, in the same PR you have a suggestion (https://github.com/ChainSafe/gossamer/pull/2827#discussion_r973318034) to remove a variable creation which is getting used twice and makes the code readable. The code that you have suggested there, I find it uglier.

Reply:
Still unneeded delta. If you had to change the code for another reason and you want to change the style, sure go ahead. But this is isolated and unneeded.

Also ideally in the future, we *should* define the config in the test and inject it to `createTestService` (and not have these obscure nil checks inside all over the place), so we'll have to define a separate variable again.

Regarding https://github.com/ChainSafe/gossamer/pull/2827#discussion_r973318034 it was not to remove a variable, but really since that was confusing to use `testVRFOutputAndProof.output` whereas you could use a more explicit `[sr25519.VRFOutputLength]byte{}`